src/Components/WeatherDetails/WeatherDetails.js

```diff
@@ -1,67 +1,107 @@
-import React from 'react';
+import React, { Component } from 'react';
 
+import Button from './WeatherShowMoreLessButton';
+import { FadeInTop } from '../../common/Animations/Animations';
+import WeatherAdditionalDetails from '../WeatherDetails/WeatherAdditionalDetails';
 import WeatherIcon from '../WeatherIcon/WeatherIcon';
 
-const WeatherDetails = ({ data }) => {
-  const { cod } = data;
-  const locationNotFound = cod !== 200 ? true : false;
+class WeatherDetails extends Component {
+  state = {
+    showAdditionalDetails: false,
+    active: false,
+  };
 
-  if (locationNotFound) {
-    return (
-      <div className="weather-details">
-        <div className="weather-details__message alert alert-danger">
-          Sorry, we didn't find the location
-        </div>
-      </div>
-    );
+  handleAdditionalDetailsState = () => {
```

Comment:
bro, just name it `showDetails = () => this.setState({showDetails: true});`

Reply:
I have rename that but it's toggle so I cant set only true value

Reply:
so add two methods

showDetails()

hideDetails()

toggling is probletaic as you're not 100% sure what the previous value could be. 
By having two methods you're always 100% sure that showDetails will set to true, while 
hideDetails will set to false